plugins/com.gwtplugins.gwt.eclipse.core/plugin.xml

```diff
@@ -706,6 +706,11 @@
             class="com.google.gwt.eclipse.core.properties.ui.GWTProjectPropertyPage"
             id="com.gwtplugins.gwt.eclipse.core.properties.ui.gwtProjectPropertyPage"
             name="General Settings">
+         <enabledWhen>
```

Comment:
This was being greedy, turning it off with a java project. I found another way to do it. 
https://github.com/gwt-plugins/gwt-eclipse-plugin/pull/223/files#diff-bf86cc2cb0c38b43ba602df9a5e590acR710

```
<enabledWhen>
   <adapt type="org.eclipse.core.resources.IProject">
      <test property="org.eclipse.core.resources.projectNature"
            value="org.eclipse.jdt.core.javanature"/>
   </adapt>         
 </enabledWhen>